app/models/compliance.rb

```diff
@@ -83,7 +83,7 @@ def self.check_compliance(target, _inputs = {})
       return
     end
 
-    compliance_result = results[:actions].nil? ? true : !results[:actions].key?(:compliance_failed)
+    compliance_result = results[:action].nil? ? true : !results[:action].key?(:compliance_failed)
```

Comment:
Note, that commit is in darga, so we probably need to backport this if @bzwei 👍  this PR.